pipeline/scripts/AddOnFunctions/violin_plots.R

```diff
@@ -0,0 +1,92 @@
+violin_plots <- function(pdf_dir, pt_name, metab_perpage, top_metab_pt=NULL) {
+
+  # set parameters for plots
+  plot_height <- 9.6 
+  plot_width <- 6
+  fontsize <- 1 
+  nr_plots_perpage <- 20
+  circlesize <- 0.8 
+  colors_4plot <- c("#22E4AC", "#00B0F0", "#504FFF","#A704FD","#F36265","#DA0641")
+  #                   green     blue      blue/purple purple    orange    red
+  
+  # patient plots, create the PDF device
+  if (grepl("Diagnostics", pdf_dir)) { prefix <- "Dx_" } else 
+    if (grepl("IEM", pdf_dir)) { prefix <- "IEM_" } else { prefix <- "R_" }
```

Comment:
For readability: add newlines between if / else blocks

Reply:
Agreed; newlines added.